micropub.php

```diff
@@ -102,11 +105,26 @@ public static function init() {
 				'description'       => 'Default Post Status for Micropub Server',
 				'sanitize_callback' => 'sanitize_text_field',
 				'show_in_rest'      => false,
-				'default'           => MICROPUB_DRAFT_MODE ? 'draft' : 'publish', // If not set can be set with deprecated property
+				'default'           => array( $cls, 'default_post_status' ),
```

Comment:
should this be `static::default_post_status()`? the docs ( https://developer.wordpress.org/reference/functions/register_setting/ ) say this should be a value, not a callable.